scripts/platforms/linkedin_apply.py

```
@@ -0,0 +1,463 @@
+    api_key=os.getenv("GROQ_API_KEY"),
+    base_url="https://api.groq.com/openai/v1"
+)
+
+
+# =========================
+# LOAD APPLICANT INFO
+# =========================
+
+with open(
+    "applicant.json",
+    "r",
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)